code/game/gamemodes/clock_cult/clock_items/clockwork_slab.dm

```diff
@@ -327,57 +329,238 @@
-		<b><font color=#DAAA18>Hierophant Network</font></b>, which allows communication to other Servants.<br>")
-		if(LAZYLEN(quickbound))
-			for(var/i in 1 to maximum_quickbound)
-				if(LAZYLEN(quickbound) < i || !quickbound[i])
-					textlist += "A <b>Quickbind</b> slot, currently set to <b><font color=#BE8700>Nothing</font></b>.<br>"
-				else
-					var/datum/clockwork_scripture/quickbind_slot = quickbound[i]
-					textlist += "A <b>Quickbind</b> slot, currently set to <b><font color=[get_component_color_bright(initial(quickbind_slot.primary_component))]>[initial(quickbind_slot.name)]</font></b>.<br>"
-		textlist += "<br>\
-		Examine the slab or swap to Recital to check the number of components it has available.<br><br>\
-		\
-		<font color=#BE8700 size=3><b><center>Purge all untruths and honor Ratvar.</center></b></font>"
+		These pages serve as the archives of Ratvar, the Clockwork Justiciar. This section of your slab has information on being as a servant, advice for what to do next, and \
+		pointers for serving the master well. You should recommended that you check this area for help if you get stuck or need guidance on what to do next.<br>")
 	return textlist.Join()
 
+//Gets text for a certain section. "Default" is used for when you first open Recollection.
+//Current sections (make sure to update this if you add one:
+//- Basics
+//- Terminology
+//- Components
+//- Scripture
+//- Power
+//- Conversion
+/obj/item/clockwork/slab/proc/get_recollection_text(section)
+	var/list/dat = list()
+	switch(section)
+		if("Default")
+			dat += "You can browse the above sections as you please. They're designed to be read in order, but feel free to pick and choose between them."
+		if("Getting Started")
+			dat += "<font color=#BE8700 size=3>Getting Started</font><br><br>"
+			dat += "Welcome, servant! This section houses the utmost basics of being a servant of Ratvar, and is much more informal than the other sections. Being a servant of \
+			Ratvar is a very complex role, with many systems, objects, and resources to use effectively and creatively.<br><br>"
+			dat += "This section of your clockwork slab covers everything that servants have to be aware of, but is a long read because of how in-depth the systems are. Knowing \
+			how to use the tools at your disposal makes all the difference between a clueless servant and a great one.<br><br>"
+			dat += "If this is your first time being a servant, relax. It's very much possible that you'll fail, but it's impossible to learn without making mistakes. For the time \
+			being, use the Hierophant Network button in the top left-hand corner of your screen to try and get in touch with your fellow servants; ignore the others for now. This button \
+			will let you send messages across space and time to any other servant in the game world. This makes it great for coordinating, and you should use it often! <i>Note:</i> Using \
```

Comment:
phrasing; "game world" is bad, and "any other servant" implies that it selects one; simplify to "all other servants"